### ISSUE TITLE
ldap group member sync problen

### ISSUE THREAD

**Issue by avelor** [mattermost/mattermost#27701]
Hello!

I've a Mattermost installation 9.6.1 and everything works pretty fine - ldap users, all member of one ldap group. Ldap filter with (memberof=thisgroup), this group is linked and configured to join all members to team.

Now I'm making new installation with same settings, now it is Mattermost Version: 9.10.0 (on 9.9.1 I've a same problem also) and group membership does not synced during user login.

So, I add user to ldap group, trying to login and got a message "you have not accessible team to join".  I check user on system console - he is added to "users", but not on linked group.
If I wait to scheduled ldap sync or force it manually - everything becomes fine.

Because user does not creates before he login, he need to login and wait ldap sync to become a member of group and team.
It's pretty strange to me and does not looks like expected behavior..

**Comment by amyblais** [comment]
@avelor What OS and version do you use? Do you see errors in the server logs? Do you have any other details to share about reproduction steps?

**Comment by avelor** [comment]
It's Ubuntu 22.04.1, docker installation. Ldap - Active Directory on Win2019. 

I tried to downgrade Mattermost to 9.6.1 with no luck..

But we've research something interesting - I started to sniff traffic on DC, and I see ldap request with groups and response during login, and I see some tcp retransmits (!) on dump (only  with group responce). So it may be some environment problem, but behavior is pretty strange (there is no retransmits then I do manual ldap sync).

we'll continue to investigate problem, of course.
